.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "semi": false,
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "jsxSingleQuote": true,
+  "bracketSpacing": true
```

Comment:
I really dislike this one - is there a particular reason for it?

Reply:
Agreed. Fixed. No reason was behind keeping it true before.